### DISCUSSION TITLE
How to get the title of the window opened in the remote server during RDP session using Freerdp

### DISCUSSION THREAD

**Discussion by srivatsagowda** [FreeRDP/FreeRDP#6937] [General]
I want to get the title of the application or window opened in remote computer during RDP session. I'm using FreeRDP client. is there a way to get the window information. I have seen function using window information including title of the window opened being implemented, but that function was not called when I created a RDP session. if not available by default, where can I make changes in source code to get the required data. please help. Thank You.

**Comment by kubistika** [comment]
It's possible only in rails mode. You can get the window title in WindowCreate/WindowUpdate callbacks if I remember correctly.

**Reply by srivatsagowda** [reply]
how do I enable rails mode?
